File: Python/Patterns/RAT(Left Incline - nums2)/Solution.py
# For an input n = 5 
# Print the RAT - left incline Pattern using numbers.
#  1
#  1 2
#  1 2 3
#  1 2 3 4
#  1 2 3 4 5

t = int(input())
while t >= 1:
    n = int(input())
    for i in range(1, n+1):
        for j in range(1,i+1):
            print(j, " ", end="")
        print()
    print()
    t -= 1

File: Java/Patterns/RAT(Left Incline - nums2)/Solution.java
import java.util.Scanner;
// For an input n = 5 
// Print the RAT - left incline Pattern using numbers.
//  1
//  2 2
//  3 3 3
//  4 4 3 4
//  5 5 5 5 5
public class Solution {
    public static void main(String[] args) {
        Scanner sc = new Scanner((System.in));
        int t = sc.nextInt();
        while (t >= 1){
            int n = sc.nextInt();
            
            // Outer Loop -> Rows
            for(int i = 1; i<=n; i++){
                // Inner Loop -> Columns
                for(int j = 1; j<=i; j++){
                    System.out.print(i + " ");
                }
                System.out.println();
            }
            t--;
            System.out.println();
        } 
        sc.close();  
    }
}
